CoreScriptsRoot/Modules/PlayerlistModule.lua

```diff
@@ -781,7 +782,27 @@ function popupHidden()
 end
 playerDropDown.HiddenSignal:connect(popupHidden)
 
+local function openPlatformProfileUI(rbxUid)
+	if not rbxUid or rbxUid < 1 then return end
+	pcall(function()
+		local platformService = game:GetService('PlatformService')
+		if platformService then
+			local platformId = platformService:GetPlatformId(rbxUid)
+			if platformId and #platformId > 0 then
+				platformService:PopupProfileUI(Enum.UserInputType.Gamepad1, platformId)
+			end
+		end
+	end)
+end
+
 local function onEntryFrameSelected(selectedFrame, selectedPlayer)
+	if isTenFootInterface then
```

Comment:
I'm a little confused on this. Do you mean we either only check isTenfootInterace or for PlatformService?